rocAL/include/api/rocal_api.h

```diff
@@ -132,4 +132,6 @@ extern "C" RocalStatus ROCAL_API_CALL rocalSerialize(RocalContext rocal_context,
  */
 extern "C" RocalStatus ROCAL_API_CALL rocalGetSerializedString(RocalContext rocal_context, char* serialized_string);
 
+extern "C" RocalContext rocalDeserialize(const char* serialized_pipeline, size_t serialized_string_size, RocalPipelineParams& pipe_params);
```

Comment:
Blocking: Please add the `ROCAL_API_CALL` annotation here (matching the definition in rocal_api.cpp). Without it this declaration is `__cdecl` on Windows while the implementation is `__stdcall`, so consumers pick up the wrong calling convention.